java/src/jmri/jmrix/mrc/MrcPacketizer.java

```diff
@@ -200,6 +200,9 @@ public void disconnectPort(MrcPortController p) {
      * enableReceiveTimeout() method), some will return zero bytes or an
      * EOFException at the end of the timeout. In that case, the read should be
      * repeated to get the next real character.
+     * @param istream data input stream from layout
+     * @return byte stream from interface
+     * @throws java.io.IOException float errors upwards
```

Comment:
We really want to explain why this is being thrown (here it can be ```@throws java.io.IOException if unable to read istream``` (I use the parameter name; this could also be "if unable to read the input stream")